pydantic/json_schema.py

```diff
@@ -1,5 +1,5 @@
-"""
-Usage docs: https://docs.pydantic.dev/2.5/concepts/json_schema/
+"""!!! abstract "Usage Documentation"
+    [Json Schema](../concepts/json_schema.md)
```

Comment:
```suggestion
    [JSON Schema](../concepts/json_schema.md)
```

(and maybe update the linked heading name if this is also formatted like this)